addons/mysql/templates/componentdefinition-mysql-5.7.yaml

```diff
@@ -0,0 +1,162 @@
+apiVersion: apps.kubeblocks.io/v1alpha1
+kind: ComponentDefinition
+metadata:
+  name: {{ include "mysql.componentDefName" . }}-5.7.44
+  labels:
+      {{- include "mysql.labels" . | nindent 4 }}
+spec:
+  provider: kubeblocks
+  description: mysql  compoent definition for Kubernetes
+  serviceKind: mysql
+  serviceVersion: 8.0.0
+
+  services:
+    - name: mysql-server
+      serviceName: mysql-server
+      spec:
+        ports:
+          - name: mysql
+            port: 3306
+            targetPort: mysql
+    - name: mysql
+      serviceName: mysql
+      generatePodOrdinalService: true
+      spec:
+        ports:
+          - name: mysql
+            port: 3306
+            targetPort: mysql
+  configs:
+    - name: mysql-replication-config
+      templateRef: oracle-mysql5.7-config-template
+      constraintRef: oracle-mysql8.0-config-constraints
+      volumeName: mysql-config
+      namespace: {{ .Release.Namespace }}
+  scripts:
+    - name: orc-mysql-scripts
+      templateRef: orc-mysql-scripts
+      namespace: {{ .Release.Namespace }}
+      volumeName: scripts
+      defaultMode: 0555
+  volumes:
+    - name: data
+      needSnapshot: true
+  systemAccounts:
+    - name: root
+      initAccount: true
+      passwordGenerationPolicy:
+        length: 10
+        numDigits: 5
+        numSymbols: 0
+        letterCase: MixedCases
+  roleProbe:
+    builtinHandler: mysql
+    periodSeconds: {{ .Values.roleProbe.periodSeconds }}
+    timeoutSeconds: {{ .Values.roleProbe.timeoutSeconds }}
+  runtime:
+    initContainers:
+      - image: {{ .Values.image.registry | default "docker.io" }}/mysql_audit_log:{{ .Values.image.tag }}
+        name: init-data
```

Comment:
the command is missing, has it been tested?

Reply:
fixed by copying the command from clusterDefination